docs/conf.py

```diff
@@ -316,18 +313,22 @@
 
 
 # Map to other F5 projects
-intersphinx_mapping = {'heat': (
-    'http://f5-openstack-heat.readthedocs.io/en/'+openstack_release.lower()+'/', None),
-    'lbaasv1': (
-    'http://f5-openstack-lbaasv1.readthedocs.io/en/1.0/', None),
+#intersphinx_mapping = {'heat': (
```

Comment:
I want to be able to use these in the future, but I have to figure out some aws auth stuff so sphinx can access the index files it needs. That's why I've commented it out instead of deleting altogether.